docker-compose.yml

```diff
@@ -1,5 +1,9 @@
 version: '3.4'
 services:
+  clamav:
+    image: mkodockx/docker-clamav
+    ports:
+      - "3310:3310"
```

Comment:
The magic is mostly here. 🪄

Reply:
likely we do not want to pull this from Dockerhub, as we may run into quota issues. Recommend setting up a repo that mirrors the clamav container to ECR, similar to the way we do here: https://github.com/department-of-veterans-affairs/vsp-infra-sentry/blob/main/.github/workflows/mirror-images.yaml and then using the container from there.

Reply:
right on, so is the idea that i would create such a yaml in vets-api/.github/workflows/mirror-images.yml?

Reply:
curious: is that an ansible file? or is that argo?

Reply:
that is a github actions workflow